src/domain/Tabs/ScrollingTabGroup/ScrollingTabGroup.tsx

```diff
@@ -69,7 +69,9 @@ export class ScrollingTabGroup extends React.Component<IScrollingTabGroupProps,
     window.addEventListener('resize', this.handleScrollUpdate)
 
     // Force an update to set the arrows correctly
-    this.forceUpdate()
+    setTimeout(() => {
+      this.forceUpdate()
```

Comment:
might need this to check `currentlyMounted` to prevent react complaining about updating unmounted components

Reply:
did not know that was a thing, neat

Reply:
hmm that is supposedly an antipatern used clearTimeout instead

Reply:
that also works :+1: